### DISCUSSION TITLE
how to use dolphinscheduler variables in seatunnel script

### DISCUSSION THREAD

**Discussion by Tomccat3** [apache/dolphinscheduler#14998] [Q&A]
1. i have a seatuunel script like this:
env {
job.mode = BATCH
}

source {
Jdbc{
password = "*"
driver = "com.mysql.cj.jdbc.Driver"
query = "SELECT id,uid,user_name,job_name,params,created_at,updated_at FROM lakeservice_task where created_at >= '"${system.biz.date}"'"
partition_column = "id"
user = "**"
}
}
sink {
console{
}
}
2. i want use dolphinscheduer varibles: ${system.biz.date}, but this param is not replaced when running.
3. are there any way to use dolphinscheduer varibles in seatunnel script?

**Comment by Tomccat3** [comment]
already solved

**Reply by kevinbds** [reply]
can you share the solution? thanks
